### DISCUSSION TITLE
Fresh retrofit version

### DISCUSSION THREAD

**Discussion by defremovSFX** [square/retrofit#3568] [General]
There've been couple of changes since may 2020 - mb it is worth to release a new version ? Especially move from databind 2.9.9 since it had ~ 10 CVEs

**Comment by JakeWharton** [comment]
We generally do not release for dependency upgrades. Quite the contrary as we want to keep our transitive dependency versions as low as possible. If you care about the version of Jackson or any other converter you should manage that version yourself in addition to the Retrofit version you depend on. Using a newer version than the one we depend on is always supported provided the library does not break binary compatibility guarantees.

The only actual code fix I see since the last release is fixing RoboVM and Java 8.

However, there's definitely a lot of stuff that I haven't been able to triage and apply potential fixes for just yet. I was on paternity leave for a few months which slowed most things down and I haven't quite got back to where I was before.
